ui/selectors/selectors.js

```diff
@@ -2454,6 +2454,10 @@ export function getHasDismissedOpenSeaToBlockaidBanner(state) {
   return Boolean(state.metamask.hasDismissedOpenSeaToBlockaidBanner);
 }
 
+export function getIsRedesignedConfirmationsFeatureEnabled(state) {
```

Comment:
Rather then adding this to the legacy `selectors.js`, can we add this to the confirmation specific selectors?

Reply:
done